tests/unit/fake_cache_api.py

```diff
@@ -45,4 +45,13 @@ def get(self, key, namespace=None):
             timed_value = namespace_dict.get(key)
         else:
             timed_value = self.cache.get(key)
-        return timed_value.get_valid_value() if timed_value else None
\ No newline at end of file
+        return timed_value.get_valid_value() if timed_value else None
+
+    def delete(self, key, namespace=None):
+        if namespace:
+            namespace_dict = self.namespaces.get(namespace)
+            if not namespace_dict:
+                return None
+            namespace_dict.pop(key, None)
+        else:
+            self.cache.pop(key, None)
```

Comment:
what is the `None` for?

Reply:
The `None` is the default value. If there is no value for the key in the cache, it'll return `None` instead of throwing an exception. This imitates the behavior of the real cache which returns `None` regardless of if it deletes anything